besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -817,6 +817,13 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final Wei txFeeCap = DEFAULT_RPC_TX_FEE_CAP;
 
+  @Option(
+      names = {"--rpc-allow-unprotected-txs"},
```

Comment:
"unprotected" is a bit of a broad term.  "Replay unprotected" is also a bit of a mouthfull.

How about `--rpc-require-eip155-txs`?  or `--rpc-require-chainid-in-txs`?

Similarly I think a more descriptive flag across the code than just "unprotected."  eip-155 is more precise but a bit harder to grock when new to the code, so `requireChainIdTXs" may be more readable.

Reply:
I believe I copied the flag from geth.

Reply:
We should do better.

Reply:
lol ok no problem.

Reply:
Going with `--rpc-require-chainid-in-txs`